qubesbuilder/plugins/publish_deb/__init__.py

```diff
@@ -149,14 +137,42 @@ def sign_metadata(self, repository_publish):
                     f"--batch --no-tty --output {release_dir / out_name} {release_dir / 'Release'}"
                 ]
                 self.log.info(
-                    f"{self.component}:{self.dist}: Signing metadata ({out_name})."
+                    f"{self.log_prefix}: Signing metadata ({out_name})."
                 )
                 executor = self.config.get_executor_from_config("publish", self)
                 executor.run(cmd)
             except ExecutorError as e:
-                msg = f"{self.component}:{self.dist}: Failed to sign metadata ({out_name})."
+                msg = (
+                    f"{self.log_prefix}: Failed to sign metadata ({out_name})."
+                )
                 raise PublishError(msg) from e
 
+    def create(self, repository_publish: str):
+        # Create skeleton
+        self.create_repository_skeleton()
+
+        # Create metadata
+        self.create_metadata(repository_publish=repository_publish)
+
+        # Sign metadata
+        self.sign_metadata(repository_publish=repository_publish)
+
+    def run(
+        self,
+        repository_publish: Optional[str] = None,
+        ignore_min_age: bool = False,
+        unpublish: bool = False,
+        **kwargs,
+    ):
+        super().run()
```

Comment:
Is this needed?

Reply:
Yes, to swallow unexpected args (`Plugin.run()` doesn't accept `unpublish` for example).